data/beaconrestapi/src/main/java/tech/pegasys/teku/beaconrestapi/handlers/v2/beacon/AbstractPostBlockV2.java

```diff
@@ -41,6 +41,7 @@ public AbstractPostBlockV2(
 
   protected AsyncApiResponse processSendSignedBlockResult(final SendSignedBlockResult result) {
     return result
+        .withChainDataResponses() // ? no metadata
```

Comment:
This was an understandable guess, but not quite :)

Basically all of the implementations extending `AbstractPostBlock` define the metadata you were looking for, so this is not required here specifically.